user_manual/changes/SB14-019.yaml

```diff
@@ -0,0 +1,8 @@
+type: bugfix
+title: Fix static evaluator on character literal
+
+description: |
+   The static evaluator were wrongly evaluating a character literal whose
+   type resolves to a type deriving the standard character type.
```

Comment:
The current wording feels twisted to me. What about: a character literal whose type derives `Standard.Character`?